src/styles/custom-components.less

```diff
@@ -79,10 +79,25 @@
         color: var(--black-700);
     }
 
+    .theme-highcontrast & {
```

Comment:
Stacks exports the `.highcontrast-mode()` mixin in `exports/mixins.less`, so let's import and use that instead